### PR TITLE
dom-api-task

### PR DESCRIPTION
**Pull request by nataliereshetnikova**
### DOM API PRACTICAL TASK

[Demo](https://nataliereshetnikova.github.io/) |
[Code base](https://github.com/nataliereshetnikova/nataliereshetnikova.github.io)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/nataliereshetnikova/js-dom/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<head>
+  <title>Primary Styles of Wine</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Wine Guide</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  <main class = container>
+    <div class = "box title-bar">
```
[the remark this answers is not in the document]
Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/index.html]: Agree - changed in the code

File: submissions/nataliereshetnikova/js-dom/script.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+},
+{
+	wineTitle: "Dessert Wine",
+	wineText: "In the mid to late 1800’s, sweet wines were more popular than dry wines. In fact, several of the most exalted wines in the world, from Sauternes in Bordeaux to Essencia from Hungary, are practically as thick as maple syrup. Dessert wines today now range from dry to sweet and are some of the boldest, most intensely flavored (and aromatic) wines in the world. There are many different types of dessert wines to explore however, if you can start with a Port or a Sauternais-styled wine (a late harvest white wine), you’ll have a great preview of what dessert wines can offer.",
+}
+];
+
+let list   = 	document.querySelector(".list");
+let title   = 	document.querySelector('.title');
+let paragraph   = 	document.querySelector('.paragraph');
```
[the remark this answers is not in the document]
Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/script.js]: Agree - changed in the code

File: submissions/nataliereshetnikova/js-dom/script.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+list.addEventListener( 'click', e => {
+	document.querySelector(".active").classList.remove("active");
+	e.target.classList.add("active");
+
+	let searchParagraph = (el, i, arr) => {
+		if(e.target.textContent == el.wineTitle)
+		return el;
+	}
+
+	showParagraph(winesObject.find(searchParagraph));
```
[the remark this answers is not in the document]
Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/script.js]: Agree - changed in the code

File: submissions/nataliereshetnikova/js-dom/index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      The 9 Primary Styles of Wine
+    </div>
+    <div class="box menu">
+      <ul class="list rolldown-list">
+        <li><a href="#" class="active" title="Sparkling Wine">Sparkling Wine</a></li>
```
[the remark this answers is not in the document]
Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/index.html]: I've previously used it in my other projects during Udemy course from Colt Steel "The Web Development Bootcamp" and the menu looks like a list to me, but still I'm not sure if it is a good practice

File: submissions/nataliereshetnikova/js-dom/script.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (AMashoshyna) [reply submissions/nataliereshetnikova/js-dom/script.js]: Oh, no, I meant previous line, after the "=" sign  -__-

File: submissions/nataliereshetnikova/js-dom/script.js [hunk 5]
```diff
@@ -1,79 +1,61 @@
...
-
-	let searchParagraph = (el, i, arr) => {
-		if(e.target.textContent == el.wineTitle)
-		return el;
-	}
-
+	document.querySelector('.active').classList.remove('active');
+	e.target.classList.add('active');
+	const searchParagraph = (el) =>
+	e.target.textContent == el.wineTitle;
```
Comment (AMashoshyna) [inline submissions/nataliereshetnikova/js-dom/script.js]: Missing indentation here: it is easier to read function body when it is indented

Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/script.js]: Thank you, it's fixed,

File: submissions/nataliereshetnikova/js-dom/script.js [hunk 6]
```diff
@@ -1,79 +1,61 @@
...
 	showParagraph(winesObject.find(searchParagraph));
-
 });
-
-//rolldown list
-// Increments the delay on each item.
+//rolldown list increments the delay on each item.
 $('.rolldown-list li').each(function () {
```
[the remark this answers is not in the document]
Comment (nataliereshetnikova) [reply submissions/nataliereshetnikova/js-dom/script.js]: Thank you for review, fixed.

**Comment by AMashoshyna** [comment]
Great job!

**Pull request merged**